### PR TITLE
add info about nomad with link to the doc in overview

### PR DESCRIPTION
**Pull request by michelle-luna**
Fixes https://github.com/circleci/circleci-docs/issues/2070

### CODE REVIEW

File: jekyll/_cci2/overview.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This article provides System Administrators with an overview of CircleCI 2.0 ins
 * TOC
 {:toc}
 
-CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. CircleCI is installable inside your private cloud or data center and is free to try for a limited time. 
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. CircleCI is installable inside your private cloud or data center and is free to try for a limited time. Contact us at [Support](https://support.circleci.com/hc/en-us) to request a trial license.
```
Comment (iynere) [inline jekyll/_cci2/overview.md]: is this really what we are supposed to tell people ? do we no longer use this ? https://circleci.com/enterprise-trial-install i send it to customers all the time cuz it's much easier than having them ask us to create & send a license

Comment (michelle-luna) [reply jekyll/_cci2/overview.md]: I just copied what was already there, I am not sure which is preferred (this is the new link to the support center), so I can change it if you think the web form is better.

File: jekyll/_cci2/overview.md [hunk 2]
```diff
@@ -27,36 +27,9 @@ There are three basic ways to install CircleCI in your environment (AWS is curre
...
-- Heroku
-
-Other cloud service deployments can be easily scripted using SSH or by installing the API client of the service with your job configuration.
-
-Contact us at [Support](https://support.circleci.com/hc/en-us) to request a trial license.
-
 ## Build Environments
 
-By default, CircleCI 2.0 Builder instances automatically provision containers according to the image configured for each job in your `.circleci/config.yml` file. 
+By default, CircleCI 2.0 Builder instances automatically provision containers according to the image configured for each job in your `.circleci/config.yml` file. Refer to the [Introduction to Nomad Cluster Operation]({{ site.baseurl }}/2.0/nomad/) to learn more about the scheduler and how to perfom basic client and cluster operations.
```
Comment (iynere) [inline jekyll/_cci2/overview.md]: this seems a little confusing..... talking about 'the scheduler' when we have never mentioned anything called a scheduler previously. i feel like there is a simple way to mention here how we use nomad on 2.0 without just linking to our nomad documentation page..... maybe just pull a couple lines from https://circleci.com/docs/2.0/nomad & paste them here, along with the link ?

File: jekyll/_cci2/overview.md (hunk 1 above)
Comment (iynere) [reply jekyll/_cci2/overview.md]: mmmm yeah i didn't realize it was already there.... either one seems fine then lol 🤷‍♀️

Comment (michelle-luna) [reply jekyll/_cci2/overview.md]: I changed it to the web form, thanks!

File: jekyll/_cci2/overview.md (hunk 2 above)
Comment (michelle-luna) [reply jekyll/_cci2/overview.md]: ok, thanks! I added a line about 'job scheduler' to make this more clear.

**Pull request merged**